### PR TITLE
Modified the button on DSA page

### LINKED ISSUE #213: Styling the button on DSA.html
## Is your feature request related to a problem?

No. It's not necessarily problem. The button on DSA.html is too basic and doesn't match the theme of the website. 

## **Describe the solution you'd like**

With just a few css styling it can be stylized without using bootstrap or any other libraries. 

Please assign this issue to me

### PR DESCRIPTION
**Pull request by cryptofox17**
# Description

I have removed the script tag for the button on DSA.html page and instead added hyperlink to index.html page. Since someone is designing the entire page I had only modified the button slightly (which I can revert back to the original if required)
Fixes #213 

## List any dependencies that are required for this change 
 NA
## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## UI /UX changes
  Before :
   
Screenshot (90)

   After :
    
Screenshot (89)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: DSA.html [hunk 1]
```diff
@@ -10,10 +10,15 @@
         }
 
         .button1 {
-            color: black;
-            font-size: medium;
-            border-radius: 8px;
-
+            color: #fff;
```
Comment (harikesh409) [inline DSA.html:13]: As per our discussion in #213 you only need to change the hyperlink so don't change these styles.

File: DSA.html [hunk 2]
```diff
@@ -24,13 +29,9 @@
     <iframe title="DSA" style="border:none" width="1400" height="635"
         src="https://whimsical.com/embed/L7xNuV2ZRSsLd6vrVtyvrQ"></iframe>
 
-    <button class="button1" onclick="goBack()">Go Back</button>
+    <button class="button1" onclick="javascript:location.href = 'index.html';">Go Back</button>
```
Comment (harikesh409) [inline DSA.html:32]: Instead of executing a javascript simply use an anchor tag.

**Review by harikesh409 — CHANGES_REQUESTED** [review]

**Comment by cryptofox17** [comment]
@harikesh409 I have made the required changes

**Comment by ravikr126** [comment]
HII @cryptofox17 CAN YOU MAKE THIS ROADMAP CENTER ALIGN. IT WILL LOOK DECENT AND TITLE  ON THE TOP ON THE ROADMAP AS  "DSA ROADMAP" align it at the center and make button at right bottom of the DSA sheet

**Comment by cryptofox17** [comment]
@ravikr126 Actually another person is working on redesigning the entire page, which is also why I have been told not to make any changes to the design.  I only had to remove script tag and instead add an anchor tag which I have done

**Review by harikesh409 — APPROVED** [review]

**Pull request merged**